en-US/GetStarted/Prototype/SetupDevice.htm

```diff
@@ -2,13 +2,13 @@
 layout: getstarted-steps-prototype
 title: Windows 10 IoT Core Official Website | Developer Resource
 description: Windows 10 IoT Core for your Internet of Things. 
-keyword: windows iot, get started, setup device, iot, intel, qualcomm, NOOBS
+keyword: windows iot, get started, home page, iot
 permalink: /en-US/GetStarted/Prototype/SetupDevice.htm
 lang: en-US
 ---
 
 <header data-grid="col-12">
-    <h1  class="c-subheading-1">Setup your device</h1>
+    <h1  class="c-subheading-1">Set up your device</h1>
```

Comment:
I think Setup is the right word.

Reply:
Rushmi recommended for us to go with the latter. I am going to run all of this content through a content writer, so there will be another PR with smoother content.